tests/unit/serve/gateway/test_gateway.py

```diff
@@ -10,12 +10,11 @@
 from jina.serve.runtimes.gateway import GatewayRuntime
 from jina.serve.runtimes.worker import WorkerRuntime
 from tests.helper import (
-    ProcessExecutor,
     _validate_custom_gateway_process,
     _validate_dummy_custom_gateway_response,
 )
-from tests.unit.yaml.dummy_gateway import DummyGateway
```

Comment:
you can't remove this, the runtime will not find `DummyGateway` when it attempts to load it.
Either keep it as it is or pass the class to the runtime (so that the class is not unused)